CHANGES.md

```diff
@@ -8,6 +8,11 @@ To be released.
 
 ### Backward-incompatible interface changes
 
+ -  `IStore.StoreStateReference<T>()` became to take the hash and index of the
+    block instead of the block itself.  [[#420]]
+     -  Therefore, it also became to lose type parameter `T`.
```

Comment:
How about this?

> `IStore.StoreStateReference<T>(string, IImmutableSet<Address>, Block<T>)` method became replaced by `StoreStateReference(string, IImmutableSet<Address>, HashDigest<SHA256>, long)` method so that it takes hash and index of a block instead of an entire `Block` object.

Reply:
👌